specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -6,33 +6,33 @@ parameters:
   "js-sdk-folder":
     default: "{cwd}/azure-sdk-for-js/"
   "csharp-sdk-folder":
-    default: "{cwd}/azure-sdk-for-csharp/"  
+    default: "{cwd}/azure-sdk-for-csharp/"
   "service-directory-name":
     default: "contosowidgetmanager"
 emit:
-  - "@azure-tools/cadl-autorest"
+  - "@azure-tools/typespec-autorest"
 options:
-  "@azure-tools/cadl-autorest":
+  "@azure-tools/typespec-autorest":
```

Comment:
@weshaggard: Do you think the `azure-resource-provider-folder` path is correct?  Should it be `../data-plane` instead, which is the correct relative path from `tspconfig.yaml` to the `data-plane` directory?

I ask because one of the rules in TypeSpec Validation is to ensure the paths of the checked-in swagger match the `azure-resource-provider-folder` property.

Reply:
I think you can use, for example, `{project-root}/../data-plane` to specify the path relative to the tspconfig.yaml, and I would recommend using {project-root} and not {cwd} throughout. {cwd} uses process current working directory and that's likely to break in scenarios that run compile from elsewhere like in the IDE. I find myself wishing {cwd} didn't exist.

Reply:
I will update to use `{project-root}`